webapp/src/main/webapp/templates/freemarker/visualization/personlevel/coAuthorPersonLevelD3.ftl

```diff
@@ -96,57 +96,82 @@ $(document).ready(function(){
             }
         });
 
+    render_chord();
+});
+
 // RENDER CHORD
 
     var labels = [];
     var uris   = [];
     var matrix = [];
+    var width  = 725;
+    var height = 725;
+    var padding = 175;
```

Comment:
Could we have the issue with responsiveness because fixing those values?

Reply:
Not sure. But it is not something new here, take a look at old lines 122-124

Reply:
ok, let's keep it as is in this PR, later we work on more responsive theme, we might improve this